06-multiple-regression.Rmd

```diff
@@ -328,17 +328,22 @@ Another way of thinking about interaction effects on teaching scores is as follo
 
 When creating regression models with one numerical and one categorical explanatory variable, we are not just limited to interaction models as we just saw. Another type of model we can use is known as a *parallel slopes* model.\index{parallel slopes model} Unlike interaction models where the regression lines can have different intercepts and different slopes, parallel slopes models still allow for different intercepts but *force* all lines to have the same slope. The resulting regression lines are thus parallel. Let's visualize the best-fitting parallel slopes model to our `evals_ch6` data.
 
-Unfortunately, the `ggplot2` package does not have a convenient way to plot a parallel slopes model. We therefore created our own special purpose function `gg_parallel_slopes()` and included it in the `moderndive` package:
+Unfortunately, the `geom_smooth()` function in the `ggplot2` package does not have a convenient way to plot parallel slopes models. We therefore created our own special purpose `geom_parallel_slopes()`\index{moderndive!geom\_parallel\_slopes()} and included it in the `moderndive` package. In other words, you won't find `geom_parallel_slopes()` in the `ggplot2` package, but rather the `moderndive` package. Thus if you want to be able to use it, you will have to ensure both the `ggplot2` and `moderndive` packages are loaded. 
```

Comment:
I think I’d prefer to give Evgeni credit here. “We” could be read as only you and I.

Reply:
Sounds good. I'll add here

Reply:
Sounds good. I’ll merge when that’s added.